src/python/pants/backend/terraform/dependency_inference.py

```diff
@@ -196,7 +196,7 @@ async def infer_terraform_module_dependencies(
     terraform_module_addresses = [
         tgt.address for tgt in candidate_targets if tgt.has_field(TerraformModuleSources)
     ]
-    return InferredDependencies(terraform_module_addresses, sibling_dependencies_inferrable=False)
+    return InferredDependencies(terraform_module_addresses)
```

Comment:
This kwarg does nothing now and was deprecated

Reply:
Thanks!